prepare-brevdata-grunnlag/src/main/java/no/nav/pensjon/brev/bestilling/pdl/ApolloClientUtils.java

```diff
@@ -0,0 +1,34 @@
+package no.nav.pensjon.brev.bestilling.pdl;
+
+import java.util.concurrent.CompletableFuture;
+
+import com.apollographql.apollo.ApolloCall;
+import com.apollographql.apollo.api.Response;
+import com.apollographql.apollo.exception.ApolloException;
+
+public class ApolloClientUtils {
```

Comment:
Siden Apollo opprinnelig ble laget for Android, så bruker det asynkron spørring, med `onResponse()` og `onFailure()` callbacks.
Dette er en hjelpeklasse for å kjøre det som en synkron spørring. Vi kan fjerne denne og bruke den asynkrone spørringen dersom vi foretrekker det.